packages/promise-kit/test/test-promise-kit.js

```diff
@@ -22,6 +22,8 @@ if (typeof globalThis.gc !== 'function') {
   engineGC = globalThis.gc;
 }
 
+/** @type {typeof rawTest} */
+// @ts-expect-error cast
```

Comment:
I wish we could type `wrapTest` correctly to avoid this.

Reply:
yeah. part of https://github.com/endojs/endo/issues/1235